features/shop/wishlist/customer/adding_product_to_wishlist.feature

```diff
@@ -0,0 +1,67 @@
+@wishlist_as_customer
+Feature: Adding a product to a wishlist
+    In order to add a product to a wishlist
+    As a customer
+    I want to be able to add products to wishlists
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Harness climbing" priced at "$30.00"
+        And the store has a product "Rope" priced at "$40.00"
+        And there is a customer "Alex Honnold" identified by an email "ahonnold@climb.com" and a password "dlonnoh1"
+        And user "ahonnold@climb.com" has a wishlist named "For Alex"
+        And there is a customer "Adam Ondra" identified by an email "aondra@climb.com" and a password "ardno1"
+        And user "aondra@climb.com" has a wishlist named "For me"
+        And I create a JWT Token for customer identified by an email "aondra@climb.com"
+
+    @graphql
+    Scenario: Adding first product to a wishlist
+        Given There is operation to add product to wishlist
+        And this operation has "id" variable with iri value of object "For me"
+        And this operation has "productVariant" variable with iri product variant "HARNESS_CLIMBING"
+        When I send that GraphQL request as authorised user
+        Then I should receive a JSON response
+        And This response body should contain:
+            | key                                                   | value             | type      |
+            | wishlist.name                                         | For me            | string    |
+            | wishlist.wishlistProducts.totalCount                  | 1                 | int       |
+            | wishlist.wishlistProducts.edges.0.node.variant.name   | Harness climbing  | string    |
```

Comment:
I think the response tables you can change to sentences i.e.: "And the total count of products is 1", "And the first variant in wishlist is Harness climbing".